_source/logzio_collections/_security-sources/sophos.md

```diff
@@ -108,3 +118,122 @@ The logs will require customized parsing so they can be effectively mapped in Ki
 
 
 </div>
+
+</div>
+<!-- tab:end -->
+
+
+<!-- tab:start -->
+<div id="windows">
+
+**Before you begin, you'll need**:
+
+* Sophos Intercept X Endpoint installed
+* Access to the [Sophos Central Cloud console](https://central.sophos.com/)
+* [Filebeat 7 installed](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-installation.html)
+* Terminal access to the instance running Filebeat. It is recommended to run the Sophos API script from the same instance running your Filebeat.
+
+
+
+<div class="tasklist">
+
+##### Configure Sophos to collect the Central Cloud logs
+
+Follow the official instructions provided by Sophos for [collecting Sophos Central Cloud logs from all machines](https://support.sophos.com/support/s/article/KB-000036372?language=en_US).
+
+The procedure involves using the Sophos API. Make sure that the `config.ini` used in the Sophos siem.py script is under `format = json` (this is the default setting).
+
+##### Download the Logz.io public certificate
+
+For HTTPS shipping, download the Logz.io public certificate to your certificate authority folder.
+
+Download the
+[Logz.io public certificate]({% include log-shipping/certificate-path.md %})
+to `C:\ProgramData\Filebeat\Logzio.crt`
+on your machine.
+
+
+##### Configure Filebeat
+
+Open the Filebeat configuration file (`C:\Program Files\Filebeat\filebeat.yml`) with your preferred text editor.
+
+Copy and paste the code block below, overwriting the previous contents, to replace the general configuration with the following settings:
+
+```yaml
+#... Filebeat
+filebeat.inputs:
+- type: log
+  paths:
+    - <<FILE_PATH>>
+  fields:
+    token: <<LOG-SHIPPING-TOKEN>>
+  fields_under_root: true
+  json.keys_under_root: true
+  encoding: utf-8
+  ignore_older: 3h
+
+#For version 7 and higher
+filebeat.registry.path: /var/lib/filebeat
```

Comment:
needs to be 'C:\ProgramData\Filebeat' and not /var/lib/filebeat

Reply:
Thanks Yotam, amended

Reply:
@nico-shishkin almost - the path should be inside single brackets.
`'C:\ProgramData\Filebeat'` Not `C:\ProgramData\Filebeat`

Reply:
Thanks Yotam, resolved.